src/big-bang-parser/big_bang_lexer.mll

```diff
@@ -63,6 +63,7 @@ rule read = parse
 
   | "=?"    { OPERATOR_EQUALITY }
   | "not=?" { OPERATOR_INEQUALITY }
+  | "<?"    { OPERATOR_LESS_THAN }
```

Comment:
We're using `=?` as our equality operator because we want to differentiate between that and assignment `=`.

Do we want to follow the pattern and use `<?` or keep it simple and consistent with other programming languages and go with `<`?

@phsmenon @rpglover64 @scottfsmith  @zepalmer